setting_up_firely_server/configuration/sqlserver/database_security.rst

```diff
@@ -0,0 +1,40 @@
+Database security
+=================
+
+To secure your database/data against malicious access you can add the following security layers. These security measures will have an impact on the performance of your Firely Server instance.
+
+Transparent Data Encryption (TDE)
+---------------------------------
+After enableing TDE, the data stored on disk will be encrypted, using a certificate to protect the keys used for encryption.
+This prevents copies of the database to be read properly without the certificate.
+The performance hit using this security measure will be on the database server, as the data is encrypted/decrypted during write/read to/from disk activities.
+
+`More information <https://learn.microsoft.com/en-us/sql/relational-databases/security/encryption/transparent-data-encryption?view=sql-server-ver16>`_
+
+Data Masking
+------------
+Data masking will obfuscate (parts of) database columns on a database user level. Granting 'unmasked' privileges to the application user and stricter masking to other users will limit the exposure of sensitive data while querying the database.
+This security measure should be used in conjunction with other security measures.
+The performance hit for Firely Server will be minimal, as it there should not be any masking happening for the database user configured for Firely Server.
+
+`More information <https://learn.microsoft.com/en-us/sql/relational-databases/security/dynamic-data-masking?view=sql-server-ver16>`_
+
+Row Level Security
+------------------
+Row level security makes it that database users cannot access rows they do not have permissions for. This could potentially reduce database access when using multiple instances of Firely Server with different database users, but this is not adviced as tasks like Bulk Data Export can be handled by each instance (which might have wrong privileges).
```

Comment:
I would suggest we don't support this and leave it at that.

Reply:
done